change/@ni-nimble-components-9da2496e-e544-4019-a666-d7fca3046771.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Comment:
Do you expect this breaking change to have an impact on clients?

Reply:
> Do you expect this breaking change to have an impact on clients?

Sorry, I meant this to be a patch change.